developers/bindings/index.md

```diff
@@ -692,6 +692,56 @@ It uses the `DiscoveryResultBuilder` to create the discovery result.
 The discovery service needs to provide the list of supported thing types, that can be found by the discovery service.
 This list will be given to the constructor of `AbstractDiscoveryService` and can be requested by using `DiscoveryService#getSupportedThingTypes` method.
 
+### Representation Property
+
+The name of one of the properties or configuration parameters, which discriminates the discovery result against a) other results of the same type, or b) already existing manually created Things.
+Typically this is a serial number, IP or MAC address.
+The representation property is used to auto-ignore discovery results of things that already exist in the system.
+This can happen a) if a thing has been created manually, or b) it has been discovered separately by two mechanisms e.g. by mDNS, and by NetBios, or UPnP.
```

Comment:
`This can happen, a) if a Thing...`
`or b) if the Thing has been discovered separately...` <== Makes the object of the sentence crystal clear.

Reply:
Ok. Done.